hipo_drf_exceptions/handlers.py

```diff
@@ -14,7 +14,11 @@ def get_fallback_message(exception):
     elif isinstance(exception, list):
         return get_fallback_message(exception[0])
     elif isinstance(exception, dict):
-        first_key = next(iter(exception))
+        if exception == {}:
+            # A dictionary can be empty. https://github.com/Hipo/hipo-drf-exceptions/issues/8
+            return get_fallback_message("")
+        else:
+            first_key = next(iter(exception))
```

Comment:
i think this should be handled in upper if block. If there is a list, it should look for the first non empty one.

I can't think of any case where we should return empty fallback message.

Reply:
I mean `elif isinstance(exception, list):` by upper if block.

Reply:
Nice point. What if the list is empty?

Reply:
That means there is no exception. There should not be an empty list.

Reply:
OK, We can ignore this edge case. Thank you.

Reply:
@alicertel Ready for re-review.